truenas/webui
# NAS-115787 / 22.12 / NAS-115787: Keep dashboard safe (by denysbutenko)

Automatic cherry-pick failed. Please resolve conflicts by running:

    git reset --hard HEAD~1
    git cherry-pick -x 529b4cbf89602a43e6bb28d72e4dfeb32fc98b1e

Added extra check to ignore unexist network widgets

Testing:

- Open Network page and create a new interface with type bridge
- Go to Dashboard and enable widget for newly created interface
- Go to Network and remove the bridge
- Go to Dashboard and check if its not broken


Original PR: https://github.com/truenas/webui/pull/6680
Jira URL: https://jira.ixsystems.com/browse/NAS-115787
